prime-router/src/main/kotlin/Element.kt

```diff
@@ -254,7 +254,9 @@ data class Element(
             Type.BLANK -> ""
             Type.DATE -> {
                 if (format != null) {
-                    val formatter = DateTimeFormatter.ofPattern(format)
+                    // extract Y/M/D only (Strip out the HH:mm:ss if any)
+                    val dateFormat = format.split(" ")[0]
+                    val formatter = DateTimeFormatter.ofPattern(dateFormat)
```

Comment:
I don't like this proposed fix. There are potentially date patterns that could come in that could have spaces in them between year, month, and day and this would break at that point. I think it's better to use the pattern found elsewhere in the file where we do a `parseBest` with a variable date time pattern, and then convert to either date or date time as the element demands. Please change this to match that pattern.  Thanks

Reply:
I incorporated your example code into the Element.kt.  I tested and everything is working great. Thanks.